02_configure_host.sh

```diff
@@ -37,7 +37,19 @@ fi
 # Allow ipmi to the virtual bmc processes that we just started
 sudo iptables -I INPUT -i virbr0 -p udp -m udp --dport 6230:6235 -j ACCEPT
 
+# Need to route traffic from the provisioning host.
+if [ -z $EXT_IF ]; then
+  sudo iptables -t nat -A POSTROUTING --out-interface ${EXT_IF} -j MASQUERADE
+  sudo iptables -A FORWARD --in-interface virbr0 -j ACCEPT
+fi
+
+# Need to pass the interface for the provisioning network on baremetal
+if [ -z $PRO_IF ]; then
+  brctl addif ${PRO_IF} brovc
+fi
+
 # Switch NetworkManager to internal DNS
 sudo mkdir -p /etc/NetworkManager/conf.d/
 sudo crudini --set /etc/NetworkManager/conf.d/dnsmasq.conf main dns dnsmasq
-sudo systemctl restart NetworkManager
+sudo systemctl reload NetworkManager
+sudo echo "nameserver ${ADDN_DNS}" | tee -a /etc/resolv.conf
```

Comment:
What is the purpose of this?

Reply:
I don't think this will work, because any subsequent restart of NetworkManager will overwrite it, we need to update the NM config so it's persistent, or add it to the dnsmasq overlay added for openshift?

Reply:
This was necessary _to_ work. 

After I run 02 it wipes /etc/resolv.conf -- so no name resolution can happen

Reply:
Added a comment above, if we can move the dns=dnsmasq to 05, this might solve the problem I am seeing on bare metal.